src/lab14.py

```diff
@@ -550,8 +544,11 @@ def paint(self, display_list):
         cmds.append(DrawText(self.x, self.y,
                              text, self.font, color))
 
-        paint_outline(self.node, cmds, rect)
+        if self.node.is_focused and self.node.tag == "input":
```

Comment:
Should this ideally be back-ported to the chapter in which it was introduced?

Reply:
See above: yes, but this will require some reasonably large changes so I'd rather push it on the stack for that chapter. I've added it to our chapter notes document.